ckan/logic/action/update.py

```diff
@@ -777,10 +783,151 @@ def _group_or_org_update(context, data_dict, is_org=False):
 
     if not context.get('defer_commit'):
         model.repo.commit()
+        # (canada fork only): background search index rebuilding
+        #TODO: upstream contrib!!
+        if (plugins.toolkit.asbool(plugins.toolkit.config.get('ckan.search.reindex_after_group_update', False))
+            and (group.title != current_title or group.name != current_name)):
+            # only reindex if the title or name has changed
+            action = 'reindex_group_datasets'
+            if is_org:
+                action = 'reindex_organization_datasets'
+            reindex_context = {
+                'model': model,
+                'user': user,
+                'ignore_auth': True,
+                'session': session
+            }
+            _get_action(action)(reindex_context, {'id': group.id})
```

Comment:
Only the title and name fields get copied into the search index, right? We should only trigger a reindex when one of those values changes.